packages/ui5-proxy-middleware/src/base/proxy.ts

```diff
@@ -28,7 +28,7 @@ export const ui5Proxy = (config: ProxyConfig, options?: Options, filter?: Filter
     });
     const today = new Date();
     const etag = `W/"${config.version || 'ui5-latest-' + today.getDate() + today.getMonth() + today.getFullYear()}"`;
-    const ui5Ver = config.version ? `/${config.version}` : '';
+    const ui5Ver = config.version && !config.version.startsWith('2') ? `/${config.version}` : '';
```

Comment:
That line is odd.

Reply:
@tobiasqueck 
without this, the redirect adds /2.0.0/ on the url when trying to access thre /resources, and this request then fails.
Guess that UI5 will introduce version-specific subfolders on their final destination, but at the moment (= temporary test instances) they do not exist.

Reply:
That is a yaml config, if you add a version there, then it is expected to be added to the url. If you don't then it isn't. So, the yaml config needs to just keep it empty, and must not add a 2.0.0 if the server doesn't offer the specific versioning.
That's nothing you need to "fix" in your code, it is a configuration by the user/consumer.